registrations/tests/test_registration_put.py

```diff
@@ -1,4 +1,5 @@
 from decimal import Decimal
+from distutils.util import execute
```

Comment:
I believe this import can be removed. :point_up:

Reply:
Very much so - now removed.